Sources/TranscribeCLI/main.swift

```diff
@@ -51,9 +65,8 @@ do {
     let webvttOutputFile = outputDirectory.appendingPathComponent(webvttOutputFilename)
     try Data(webvtt.utf8).write(to: webvttOutputFile)
 
-} catch let error as TranscribeCLIError {
-    print("Error: \(error.message) (\(error.code))", to: &stdError)
-    exit(Int32(error.code))
+} catch let error as ArgumentParserError {
+    print(error.description)
```

Comment:
It's important to keep the `exit` call with a non-zero argument to tell the shell that there was an error.

Reply:
you are absolutly right! I changed it in c1b5138

Reply:
Thanks.